### DISCUSSION TITLE
v3

### DISCUSSION THREAD

**Discussion by CalebQ42** [CalebQ42/SWAssistant#17] [Announcements]
I'M BACK.
This is currently beta and has some limitations compared to the old versions:

- No Google Drive saving.
- No Downloads.
- Only available in English.

Some of the new stuff:

- New codebase that should be a lot easier to maintain.
- Better and less confusing UI.
- UI should handle large screens A LOT better. In particular, GM Mode should be more consistently usable.

<hr /><em>This discussion was created from the release <a href='https://github.com/CalebQ42/SWAssistant/releases/tag/'>v3</a>.</em>

**Comment by RebelScum126** [comment]
It'd be nice to be able to add pictures for character art and vehicles.

**Reply by CalebQ42** [reply]
Sorry I didn't get back to you earlier. It's definitely something I want to add at some point, but it's not something I particularly want to work on at the moment.
